manta-pay/Cargo.toml

```diff
@@ -40,7 +40,10 @@ required-features = ["clap", "groth16", "simulation"]
 # Enable Arkworks Backend
 arkworks = [
     "ark-bls12-381",
+    "ark-ec",
     "ark-ed-on-bls12-381",
+    "ark-ff",
+    "ark-serialize",
```

Comment:
These are accessed through `manta_crypto::arkworks::ec` or `manta_crypto::arkworks::ff`, etc. and don't need to be imported anymore.